tasks/components/SignInButton.tsx

```diff
@@ -1,32 +0,0 @@
-import type { Languages } from '@helpwave/common/hooks/useLanguage'
-import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
-import { useTranslation } from '@helpwave/common/hooks/useTranslation'
-import { getAuthorizationUrl } from '../utils/oauth'
-import { Button } from './Button'
-
-type SignInButtonTranslation = {
-  signInWithHelpwave: string
-}
-
-const defaultSignInButtonTranslation: Record<Languages, SignInButtonTranslation> = {
-  en: {
-    signInWithHelpwave: 'Sign in with helpwave',
-  },
-  de: {
-    signInWithHelpwave: 'Anmelden mit helpwave',
-  },
-}
-
-const SignInButton = ({
-  language,
-}: PropsWithLanguage<SignInButtonTranslation>) => {
-  const translation = useTranslation(language, defaultSignInButtonTranslation)
-
-  return (
-    <Button variant="secondary" onClick={() => {
-      getAuthorizationUrl().then((url) => window.location.assign(url))
```

Comment:
Can you add this to the `onClick` handler on `pages/auth/index.tsx`? When this pr introduces changes to the current SignInButton, the behavior should stay the same.